src/scss/components/_signature.scss

```diff
@@ -2,20 +2,37 @@
 
 @import '../mixins/bem';
 
-.signature-pad-body {
-  @include margin(auto, $value-mobile: 0.5em);
-  width: 100%;
-  height: 150px;
-  padding: 0;
-  margin: 0;
-
-  @include element('image') {
+@mixin signature($color-border-focus: null, $color-border-focus-visible: null) {
+  .signature-pad-body {
+    @include margin(auto, $value-mobile: 0.5em);
     width: 100%;
-    display: none;
+    height: 150px;
+    padding: 0;
+    margin: 0;
+
+    @include element('image') {
+      width: 100%;
+      display: none;
+    }
+
+    .fa-refresh {
+      // accessibility focus
+      &:focus,
+      &:focus-visible {
+        // outline: 2px solid var(--of-color-info);
+        // TODO: can be removed once there are utrecht design tokens for the bottom/right border color
+        // issue: https: //github.com/nl-design-system/utrecht/issues/1406
+        border-color: $color-border-focus !important;
+        border-right-color: $color-border-focus !important;
+        border-bottom-color: $color-border-focus !important;
```

Comment:
* I don't think the `!important` is required here
* the `$color-border-focus` variable does not seem assigned anywhere, I'd rather just directly add/use a component specific design token: `{of.signature.reset-icon.focus.border-color}`

Reply:
Yes @sergei-maertens I will attempt that + 
+ I do wonder if there is a reason why Utrecht uses the correct order "**border-color**" but OF uses a reverse "**color-border**" which would be incorrect as an actual CSS attribute.

Reply:
no particular reason - we're coming from sass variables that "grouped" all variables under the `color` namespace, while NLDS favours actual CSS attributes as token part names. OF has the base design token `{of.color.border}`, so we just reflect that structure and map to the CSS properties.